### DISCUSSION TITLE
Is Harbor v2.10.0 compatible with external database with postgres version 13.13 ?

### DISCUSSION THREAD

**Discussion by NagenderPulluri** [goharbor/harbor#20091] [General]
Is Harbor v2.10.0 compatible with external database with postgres version 13.13 ?
and when can we expect v2.11.0 version of Harbor ?

**Comment by MinerYang** [comment]
We are using postgres (PostgreSQL) 14.10 for Harbor v2.10.0
And we didn't do compatibility test on PG13 on harbor v2.10
Will expect PG15 in v2.11
